WINDOW_SFML/Mouse_SFML.cpp

```diff
@@ -0,0 +1,57 @@
+#include "Mouse_SFML.h"
+
+Mouse_SFML::Mouse_SFML()
+{
+	_fromNative[sf::Mouse::Left] = Mouse::Left;
+	_fromNative[sf::Mouse::Right] = Mouse::Right;
+	_fromNative[sf::Mouse::Middle] = Mouse::Middle;
+	_fromNative[sf::Mouse::XButton1] = Mouse::XButton1;
+	_fromNative[sf::Mouse::XButton2] = Mouse::XButton2;
+
+
+	_toNative[Mouse::Left] = sf::Mouse::Left;
+	_toNative[Mouse::Right] = sf::Mouse::Right;
+	_toNative[Mouse::Middle] = sf::Mouse::Middle;
+	_toNative[Mouse::XButton1] = sf::Mouse::XButton1;
+	_toNative[Mouse::XButton2] = sf::Mouse::XButton2;
+}
+
+sf::Mouse::Button Mouse_SFML::toNative(Mouse::Button key)
+{
+	return _toNative[key];
+}
+
+Mouse::Button Mouse_SFML::fromNative(sf::Mouse::Button key)
+{
+	return _fromNative[key];
+}
+
+sf::Mouse::Wheel Mouse_SFML::toNative(Mouse::Wheel key)
+{
+	if (key == Mouse::HorizontalWheel)
+		return sf::Mouse::HorizontalWheel;
+	return sf::Mouse::VerticalWheel;
+}
+
+Mouse::Wheel Mouse_SFML::fromNative(sf::Mouse::Wheel key)
+{
+	if (key == sf::Mouse::HorizontalWheel)
+		return Mouse::HorizontalWheel;
+	return Mouse::VerticalWheel;
+}
+
+bool Mouse_SFML::isButtonPressed(Mouse::Button button)
+{
+	return sf::Mouse::isButtonPressed(toNative(button));
+}
+
+Vector2i Mouse_SFML::getPosition()
```

Comment:
Well yeah that makes sense. But what's the type of the Window needed?  Because it might break the abstraction. I'll see tomorrow, but if you want please fix it